packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -261,4 +305,208 @@ export default class metadataBackup {
       }),
     ])
   }
+
+  // {
+  //   [<Remote ID>]: [{
+  //     id: `${remoteId}/folderPath`,
+  //     jobId,
+  //     jobName,
+  //     scheduleId,
+  //     scheduleName,
+  //     timestamp
+  //   }]
+  // }
+  async listXoMetadataBackups(remoteIds: string[]) {
+    const app = this._app
+    const backupsByRemote = {}
+
+    await Promise.all(
+      remoteIds.map(async remoteId => {
+        try {
+          const handler = await app.getRemoteHandler(remoteId)
+          const getListFromPath = createBackupsListGetter(
```

Comment:
This is really unclear, too meta for me.

Either find a better name or a better (design) pattern.

Reply:
`createBackupsListGetter` -> `getBackupsListFactory`, `getListFromPath` => `getBaackupsList` ?